src/test/java/core/basesyntax/service/ReaderServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReaderServiceTest {
+    private static ReaderService readerService;
+
+    @BeforeClass
+    public static void init() {
+        readerService = new ReaderServiceImpl();
+    }
+
+    @Test
+    public void readFromFile_Ok() {
+        String fileName = "src/test/java/core/basesyntax/recourses/input-test.csv";
```

Comment:
```suggestion
        String fileName = "src/test/recources/input-test.csv";
```
same for all test resourCes